lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -102,19 +106,31 @@ qx.Class.define("callbackery.ui.plugin.Action", {
         _populate: function(cfg,buttonClass,getFormData){
             var tm = this._tableMenu = new qx.ui.menu.Menu;
             var menues = {};
+            let plugin = this._plugin;
             cfg.action.forEach(function(btCfg){
-                var button;
+                var button, menuButton;
                 var label = btCfg.label ? this.xtr(btCfg.label) : null;
-                var menuButton;
                 switch (btCfg.action) {
                     case 'menu':
                         var menu = menues[btCfg.key] = new qx.ui.menu.Menu;
                         if (btCfg.addToMenu != null) { // add submenu to menu
-                            menues[btCfg.addToMenu].add(new qx.ui.menu.Button(label, null, null, menu));
+                            button = new qx.ui.menu.Button(label, null, null, menu)
+                            menues[btCfg.addToMenu].add(button);
                         }
                         else { // add menu to form
-                            this.add(new qx.ui.form.MenuButton(label, null, menu));
+                            button = new qx.ui.form.MenuButton(label, null, menu);
+                            this.add(button);
+                        }
+//                        console.log('adding button to menu, key=', btCfg.key);
+                        let btnId = btCfg.key + 'Button'
+                        let oldButton = qx.core.Id.getInstance().getQxObject(cfg.name + '/' + btnId);
+                        if (oldButton) {
+                            plugin.removeOwnedQxObject(btnId);
+                            oldButton.destroy();
                         }
```

Comment:
this seems odd, that 'old' buttons are hanging around here ...

Reply:
Removed